resources/intent/weedo_x40/Extrudr/weedo_x40_0.4_Extrudr_GreenTECPro_Red_Draft_Draft.inst.cfg

```diff
@@ -0,0 +1,22 @@
+[general]
+version = 4
+name = Quick
+definition = weedo_x40
+
+[metadata]
+setting_version = 17
+type = intent
+quality_type = draft
+intent_category = quick
+material = Extrudr_GreenTECPro_Black_175
```

Comment:
The ID of the profile says _greentecpro_red, but the material is greentecpro_black

I think it's better to just add greentecpro as a generic material and let all of them reference to that